pkg/server/config.go

```diff
@@ -268,16 +271,32 @@ func NewConfig(opts *kcpserveroptions.CompletedOptions) (*Config, error) {
 		c.userToken = userToken
 	}
 
-	if err := opts.GenericControlPlane.Audit.ApplyTo(c.GenericConfig); err != nil {
+	bootstrapKcpConfig := rest.CopyConfig(c.identityConfig)
+	bootstrapKcpConfig.Impersonate.UserName = "system:kcp:bootstrapper"
+	bootstrapKcpConfig.Impersonate.Groups = []string{bootstrappolicy.SystemKcpWorkspaceBootstrapper}
+	bootstrapKcpConfig = rest.AddUserAgent(bootstrapKcpConfig, "kcp-identity-bootstrapper")
```

Comment:
why kcp-identity-bootstrapper ? Is there a non-identity one?

Reply:
ic, what you mean. Not sure it is helpful. If I read `kcp-identity-bootstrap` in my logs I wouldn't know what it means.

Reply:
do you want to share the same agent w/everything else?